code/game/objects/items/syndielad.dm

```diff
@@ -0,0 +1,300 @@
+#define SYNDIELAD-POWEROFF "poweroff"
+#define SYNDIELAD-UPLINK "uplink"
+#define SYNDIELAD-PINPOINTER "pinpointer"
+#define SYNDIELAD-DETONATE "detonate"
```

Comment:
it doesn't like these

Reply:
remove the `SYNDIELAD-` before them

Reply:
Better solution is to replace `-` with `_`. Without the prefix, these are all generic define names that could cause conflicts at some point.